src/Clock.tsx

```diff
@@ -0,0 +1,55 @@
+import { Component } from 'react';
+
+type Props = {
+  name: string,
+};
+
+type State = {
+  time: string,
+};
+
+export class Clock extends Component<Props, State> {
+  state: Readonly<State> = {
+    time: (new Date()).toUTCString().slice(-12, -4),
+  };
+
+  timerId = 0;
+
+  componentDidMount() {
+    this.timerId = window.setInterval(() => {
+      this.setState({ time: (new Date()).toUTCString().slice(-12, -4) });
+      // eslint-disable-next-line no-console
+      console.info(this.state.time);
+    }, 1000);
+  }
+
+  componentDidUpdate(prevProps: Readonly<Props>) {
+    if (prevProps.name !== this.props.name) {
+      // eslint-disable-next-line no-console
+      console.debug(`Renamed from ${prevProps.name} to ${this.props.name}`);
+    }
+  }
+
+  componentWillUnmount() {
+    window.clearInterval(this.timerId);
+  }
+
+  render() {
+    const { name } = this.props;
+    const { time } = this.state;
+
+    return (
+      <div className="Clock">
+        <strong className="Clock__name">
```

Comment:
Where are all the style files?

Keep in mind, all the components (exclude App.tsx) should be in "components" folder, there should be a separate folder for each component and his style